mrn_mysql_compat.h

```diff
@@ -906,6 +906,10 @@ using TABLE_LIST = Table_ref;
                          to))
 #endif
 
+#if defined(MRN_MARIADB_P) && (MYSQL_VERSION_ID >= 110400)
```

Comment:
The condition is the same as line 886, but is it better to separate it so that it is easier to understand?

Reply:
I summarized it because it was redundant.